### PR TITLE
Use golang 1.20 for binary test code coverage

### PR DESCRIPTION
**Pull request by adrianreber**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -16,18 +16,32 @@ VERSION := $(VERSION_MAJOR)$(if $(VERSION_MINOR),.$(VERSION_MINOR))$(if $(VERSIO
...
+		exit 0 ;\
+	elif [ $(GO_MAJOR_VER) -lt $(MIN_GO_MAJOR_VER) ]; then \
+		echo '$(GO_VALIDATION_ERR)';\
+		exit 1; \
+	elif [ $(GO_MINOR_VER) -lt $(MIN_GO_MINOR_VER) ] ; then \
+		echo '$(GO_VALIDATION_ERR)';\
+		exit 1; \
+	fi
+
 all: $(NAME)
```
Comment (rst0git) [inline Makefile:36]: The `all` target should appear before `check-go-version`.

**Comment by rst0git** [comment]
@adrianreber It might be good to add `.coverage/` and `checkpointctl.coverage` to gitignore

File: Makefile (hunk 1 above)
Comment (adrianreber) [reply Makefile]: ah, good point, thanks, done

**Comment by adrianreber** [comment]
> @adrianreber It might be good to add `.coverage/` and `checkpointctl.coverage` to gitignore

added

**Review by rst0git — APPROVED** [review]
LGTM

**Pull request merged**